src/Mapping/Driver/DatabaseDriver.php

```diff
@@ -560,4 +570,14 @@ private static function getReferencedColumnNames(ForeignKeyConstraint $foreignKe
 
         return $foreignKey->getForeignColumns();
     }
+
+    /** @return string[] */
+    private static function getIndexedColumns(Index $index): array
+    {
+        if (! method_exists(Index::class, 'getIndexedColumns')) {
```

Comment:
I'd invert this. If `getIndexedColumns()` exists, call it. Otherwise, call the deprecated method.

Reply:
done